final_task/rss_reader/Handler.py

```diff
@@ -0,0 +1,73 @@
+import json
+import logging
+
+import feedparser
+from Entry import Entry
+from Logging import logging_decorator
+
+
+class Handler:
+    """class for handling different options: --version, --json, --limit Limit"""
+    @logging_decorator
+    def __init__(self, source: str, limit: int, version: float):
+        self.source = source
+        self.limit = limit
+        self.version = version
+        self.parsed = feedparser.parse(self.source)
+        logging.info("Handler object created")
+
+    """options of command line:"""
```

Comment:
Usually docstrings are ituated only at the beginning of the function/class/module. Here specifically it is better to make that a comment

Reply:
fixed

Reply:
And I have a question about the point 4 above:
<img src="" width="130" height="86" alt="See Photos of the SpeedKore Dodge Charger" align="left" title="See Photos of the SpeedKore Dodge Charger" border="0" >
this is a string from url "https://news.yahoo.com/rss/"
And this image isn't on the html version of this page. What can I do in this case?

Reply:
Well, that is still strange, but in this case, when you have no text and no image it seems logical to me to ignore this piece of news in the feed

Reply:
In such articles text exists but img tags has no attribute src